README.md

```diff
@@ -72,12 +78,43 @@ followed by:
 $ heroku rake db:migrate
 ```
 
-To see updated (or data in new fields) data on heroku, you need to run:
+To see updated (or data in new fields) data on heroku, you need to run all of the import tasks:
 
 ```bash
-$ heroku rake import:organisation[{department-slug}]
+$ heroku rake import:all_organisations
+$ heroku rake import:content_items_by_organisation[{department-slug}]
+$ heroku rake import:number_of_views_by_organisation[{department-slug}]
 ```
 
+### Set up Google Analytics credentials in development
+
+To be able to import data from Google Analytics, a few environment variables need to be set.
+
+1) Create files `config/local_env.yml`
```

Comment:
I have created a [Trello card](https://trello.com/c/hdoeQ1zT/111-use-rails-configuration-for-environment-variables) so we explore two things:

1. Provide a `config/local_env.yml.copy` with the template including all environment variables and update the readme to advise the reader to copy to `config/local_env.yml` and fill the environment variables.
2. Use `Rails.configuration` object to access environment variables within the app.
3. Set a default value for CPM_GOVUK_VIEW_ID